cmd/crictl/image.go

```diff
@@ -142,6 +144,11 @@ var listImageCommand = &cli.Command{
 			Aliases: []string{"q"},
 			Usage:   "Only show image IDs",
 		},
+		&cli.StringSliceFlag{
+			Name:    "filter",
+			Aliases: []string{"f"},
+			Usage:   "The filtering flag format is of 'dangling=(true/false)', 'reference=regex', '(before|since)=<image-name>[:<tag>]|<image id>|<image@digest>'",
```

Comment:
This could be made such that it explains what the filter does, and then how to use it.